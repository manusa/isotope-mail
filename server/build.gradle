buildscript {
	ext {
		springBootVersion = '2.0.4.RELEASE'
	}
	repositories {
		mavenCentral()
	}
	dependencies {
		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
	}
}

apply plugin: 'java'
apply plugin: 'eclipse'
apply plugin: 'org.springframework.boot'
apply plugin: 'io.spring.dependency-management'

group = 'com.marcnuri.isotope'
version = '0.0.1-SNAPSHOT'
sourceCompatibility = 1.8

repositories {
	mavenCentral()
}


dependencies {
	compile('org.springframework.boot:spring-boot-starter')
	compile('org.springframework.boot:spring-boot-starter-web')
	compile('org.springframework.boot:spring-boot-starter-hateoas')
	compile('com.sun.mail:javax.mail:1.6.1')
	testCompile('org.springframework.boot:spring-boot-starter-test')
	testCompile('com.marcnuri:spring-common-test:0.0.2')
}
