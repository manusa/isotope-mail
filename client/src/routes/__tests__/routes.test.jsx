import React from 'react';
import {shallow} from 'enzyme/build';
import Routes from '../routes';


describe('Routes component test suite', () => {
  test('Snapshot render', () => {
    // When
    const routes = shallow(<Routes />);
    // Then
    expect(routes).toMatchSnapshot();
  });
});
