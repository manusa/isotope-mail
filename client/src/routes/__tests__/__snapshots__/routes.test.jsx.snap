// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes component test suite Snapshot render 1`] = `
<Router
  basename="/"
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Route
      exact={true}
      path="/configuration-not-found"
      render={[Function]}
    />
    <Connect(ApplicationReadyRoute)
      exact={true}
      path="/login"
      render={[Function]}
    />
    <Connect(ApplicationReadyRoute)
      component={[Function]}
      exact={true}
      path="/"
    />
  </Switch>
</Router>
`;
